pkgs/desktops/gnome/extensions/buildGnomeExtension.nix

```diff
@@ -0,0 +1,54 @@
+{ pkgs, lib, stdenv, fetchzip, xxd }:
+
+let
+
+  buildGnomeExtension = {
+    # Every gnome extension has a UUID. It's the name of the extension folder once unpacked
+    # and can always be found in the metadata.json of every extension.
+      uuid
```

Comment:
BTW, with indented attrs, you might want to use trailing commas: https://github.com/nix-community/nixpkgs-fmt/issues/248

Reply:
https://github.com/NixOS/nixpkgs/pull/108414#discussion_r598357209

Reply:
You would typically use `{` at the same level as the commas as currently used by nixpkgs-fmt, or indented with trailing commas as suggested in https://github.com/nix-community/nixpkgs-fmt/issues/248. As is, the style is somewhere in between.

Reply:
Better now?